src/BuildMetadata.ts

```diff
@@ -21,6 +21,9 @@ export class BuildMetadata {
     private static readonly MINOR_VERSION_MULTIPLIER = 100;
     private static readonly PATTERN = /^([\d]+)\.([\d]+)(?:\.([\d]+))?(-[\w]+)?(-SNAPSHOT)?(-BETA-.)?$/;
 
+    /* tslint:disable:member-ordering */
```

Comment:
Why do we need to disable `member-ordering`? This line can go just below UNKNOWN_VERSION_ID.

Reply:
No, it cannot because `MAJOR_VERSION_MULTIPLIER ` isn't declared there.